lib/FieldType/FieldTypeFormMapperDispatcher.php

```diff
@@ -25,10 +25,20 @@ class FieldTypeFormMapperDispatcher implements FieldTypeFormMapperDispatcherInte
     /**
      * FieldType form mappers, indexed by FieldType identifier.
      *
-     * @var FieldTypeFormMapperInterface[]
+     * @var FieldTypeFormMapperInterface[]|FieldValueFormMapperInterface[]|FieldDefinitionFormMapperInterface[]
```

Comment:
Why this change, when in addMapper() below it looks like only `FieldTypeFormMapperInterface` will ever be used here?